coverage/xmlreport.py

```diff
@@ -9,12 +9,13 @@
 import sys
 import time
 import xml.dom.minidom
-
+import numpy as np
```

Comment:
Seems like this will add a dependency on `numpy` to `coverage`.  Is that really necessary for improved reporting?

And it seems like the changes break the test environment.

Reply:
This is my fault. Excuse-me. I will remove it.

Reply:
@nedbat I noticed that py coverage doesn't assign the methods to their classes. So, I made some modifications to generate the following output:

<package name="tested_module" line-rate="0.8571" branch-rate="0" complexity="0">
			<classes>
				<class name="Department" filename="tested_module/company_model.py" complexity="0" class_lines="23" class_hits="18" line-rate="0.7826">
					<method name="__init__" method_lines="4" method_hits="4" method_misses="0">
						<line number="4" hits="1"/>
						<line number="5" hits="1"/>
						<line number="6" hits="1"/>
						<line number="7" hits="1"/>
					</method>
					<method name="__str__" method_lines="1" method_hits="0" method_misses="1">
						<line number="10" hits="0"/>
					</method>
					<method name="add_member" method_lines="2" method_hits="2" method_misses="0">
						<line number="13" hits="1"/>
						<line number="14" hits="1"/>
					</method>
					<method name="remove_member" method_lines="1" method_hits="0" method_misses="1">
						<line number="17" hits="0"/>
					</method>
					<method name="count" method_lines="1" method_hits="1" method_misses="0">
						<line number="21" hits="1"/>
					</method>
					<method name="__len__" method_lines="1" method_hits="0" method_misses="1">
						<line number="24" hits="0"/>
					</method>
				</class>
				<class name="CompanyModel" filename="tested_module/company_model.py" complexity="0" class_lines="22" class_hits="17" line-rate="0.7727">
					<method name="__init__" method_lines="3" method_hits="3" method_misses="0">
						<line number="29" hits="1"/>
						<line number="30" hits="1"/>
						<line number="31" hits="1"/>
					</method>
					<method name="__str__" method_lines="1" method_hits="0" method_misses="1">
						<line number="34" hits="0"/>
					</method>
					<method name="add_user" method_lines="6" method_hits="5" method_misses="1">
						<line number="37" hits="1"/>
						<line number="38" hits="1"/>
						<line number="39" hits="1"/>
						<line number="40" hits="1"/>
						<line number="41" hits="1"/>
						<line number="42" hits="0"/>
					</method>
					<method name="count_persons_per_department" method_lines="1" method_hits="1" method_misses="0">
						<line number="45" hits="1"/>
					</method>
					<method name="get_headcount_for" method_lines="1" method_hits="1" method_misses="0">
						<line number="48" hits="1"/>
					</method>
				</class>
			</classes>
		</package>
	</packages>

Reply:
@nedbat You are right. I didn't commit the modification on the test suite. I will make it again. Excuse me for that, I live in Brazil and we suffered electrical issue and my HD was damaged. I think that this merge commit was complete. Again, sorry about this noob action.